### PR TITLE
Add support for configuring point size in Visualize Lidar GUI plugin

### PR DESCRIPTION
**Pull request by iche033**
# 🎉 New feature

depends on https://github.com/ignitionrobotics/ign-rendering/pull/392

## Summary

Added a spin box for configuring size of lidar point visualization in `Visualize Lidar` plugin. Note it only affects `Point` visualization type. Works for `ogre` and `ogre2` render engine.

## Test it

1. Launch an example world with gpu ray sensor, e.g.:

    ```
    ign gazebo -v 4 gpu_lidar_sensor.sdf
    ```

1. Open the `Visualize Lidar` GUI plugin by going to the top right menu (3 vertical dots), scroll down and click on `Visualize Lidar`.

1. Unpause the simulation by hitting the play button
1. Refresh the topic drop down list and select the `/lidar` topic
1. Select `Points` Visual Type to see the default point visualization with size `1`
1. Change `Point Size` to `5` and hit Enter to see larger points.

lidar_point_size

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] `codecheck` passed (See [contributing](https://ignitionrobotics.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+repo%3Aosrf%2Fsdformat+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge**

### CODE REVIEW

File: src/gui/plugins/visualize_lidar/VisualizeLidar.qml [hunk 1]
```diff
@@ -131,4 +131,21 @@ GridLayout {
...
+
+  IgnSpinBox {
+    Layout.columnSpan: 2
+    id: pointSize
+    maximumValue: 1000
+    minimumValue: 1
+    value: 1
+    onEditingFinished: VisualizeLidar.UpdateSize(pointSize.value)
+  }
+
```
Comment (ahcorde) [inline src/gui/plugins/visualize_lidar/VisualizeLidar.qml:150]:
```suggestion
```

**Review by ahcorde — APPROVED** [review]
LGTM, just a minor style fix

File: src/gui/plugins/visualize_lidar/VisualizeLidar.qml (hunk 1 above)
Comment (iche033) [reply src/gui/plugins/visualize_lidar/VisualizeLidar.qml]: c0296add5b12b88874f22cb1e228012ddf6b2a1e

**Comment by ahcorde** [comment]
cpplint is failing. Fixed here https://github.com/ignitionrobotics/ign-gazebo/pull/1025

**Review by WilliamLewww — APPROVED** [review]

**Pull request merged**